media-source-respec.html

```diff
@@ -2258,32 +2192,26 @@
       <h2>SourceBufferList Object</h2>
       <p>SourceBufferList is a simple container object for <a>SourceBuffer</a> objects. It provides read-only array access and fires events when the list is modified.</p>
 
-      <dl title="interface SourceBufferList : EventTarget" class="idl">
-        <dt>readonly attribute unsigned long length</dt>
-        <dd>
+      <div><pre class="idl">interface SourceBufferList : EventTarget {
```

Comment:
Ditto (many types missing links here, in this object's IDL and method details...)